include/deal.II/base/exceptions.h

```diff
@@ -456,7 +456,7 @@ namespace deal_II_exceptions
       out << "    " << arg << std::endl;                                  \
     }                                                                     \
   private:                                                                \
-    const std::string arg;                                                \
+    const std::string &arg;                                               \
```

Comment:
I don't understand this change. How does this result in anything but a dangling reference to a temporary as the exception propagates?

Reply:
huh, interesting point. I guess we don't have a problem if we call abort in ``Assert()``, but I think you are correct in the case we store and throw the exception. I guess this means we need to make the copy. Too bad.